app/src/app/components/providers/ThemeProvider.tsx

```diff
@@ -26,17 +26,14 @@ export const ThemeProvider = ({ children, ...props }: ThemeProviderProps) => {
     () => (localStorage.getItem(LocalStorageKeys.THEME) as Theme) || DEFAULT_THEME
   );
 
-  const availableThemes = useMemo((): Theme[] => {
-    if (isLoading || !brandingConfig) {
-      return [];
-    } else if (brandingConfig.darkModeEnabled && !brandingConfig.lightModeEnabled) {
-      return ["dark"];
-    } else if (brandingConfig.lightModeEnabled && !brandingConfig.darkModeEnabled) {
-      return ["light"];
-    } else {
-      return DEFAULT_AVAILABLE_THEMES;
-    }
-  }, [brandingConfig, isLoading]);
+  let availableThemes = DEFAULT_AVAILABLE_THEMES;
+  if (isLoading || !brandingConfig) {
+    availableThemes = [];
```

Comment:
Can you please pull out into a function so we don't have to make availableThemes mutable and reassign.... 
`const availableThemes = getAvailableThemes(brandingConfig)`